docs/src/enterprise/getstarted/install.md

```diff
@@ -37,13 +38,216 @@ Access to configure your SSO IdP [supported by lakeFS Enterprise][lakefs-sso-ent
+    > **Note**: Instead of configuring the license token content via the lakeFS Enterprise configuration file, you can set it via the environment variable: `LAKEFS_LICENSE_CONTENTS`.
+
+    !!! warning
+        You must use exactly one method, providing both or neither will cause the server to fail to start.
+
+??? info "Licensed Features"
+    The following lakeFS Enterprise features must be included in the lakeFS Enterprise license in order to be available:
+
+    **Authentication & Authorization**
+
+    - **[SSO (Single Sign-On)](../../security/sso.md)**
+    - **[RBAC (Role-Based Access Control)](../../security/rbac.md)**
+    - **[SCIM (System for Cross-domain Identity Management)](../../howto/scim.md)**
+    - **[IAM (Identity and Access Management) Role Authentication](../../security/external-principals-aws.md)**
+
+    **Advanced Functionality**
+
+    - **[Mount](../../reference/mount.md)**
+    - **[Metadata Search](https://info.lakefs.io/metadata-search)**
+    - **[MSB (Multiple Storage Backends)](../../howto/multiple-storage-backends.md)**
+    - **[Transactional Mirroring](../../howto/mirroring.md)**
+    - **[Sparkless GC (Garbage Collection)](../../howto/garbage-collection/standalone-gc.md)**
+
+    !!! warning 
+        Without a license for these features, they will be disabled, and any attempt to access them will result in a 'feature not licensed' error.
+
+??? info "License Monitoring & Updates"
+    **Updating Your License**
+
+    _When Using File Path:_
+
+    1. Replace the content of your license file with the new license token.  
+    2. lakeFS Enterprise will automatically detect and reload the new license within 1 minute.  
+    
+    !!! warning
+        You cannot change the file path itself to point to a new file while the server is running.
+
+    _When Using Direct Contents:_
+
+    1. Update your configuration file with the new license token.
+    2. Restart the lakeFS Enterprise server to apply changes.
+
+    **Automatic Monitoring**
+
+    - **Validation and Expiry Check**: Occurs every 24 hours to verify license validity and expiry.
+    - **File Monitoring**: When using `license.path`, the server checks every minute to detect license file changes so that when the license token gets updated in the file given in the path, there is no need to restart the server and the license updates automatically.
+
+    **License Expiry Server Behavior**
+
+    - **31+ days before expiry**: No warnings.
+    - **30 to 0 days before expiry**: A warning notification appears in the web UI.
+    - **1 to 30 days After expiry**: An error notification appears in the web UI and logs, but the server continues running.
+    - **31+ days after expiry**: The server will shut down and refuse to start.
+
+??? info "Reading License Token via API"
+
+    **Request:**
+    ```bash
+    GET https://your-lakefs-server/api/v1/license
+    ```
+
+    **Response:**  
+    ```json
+    {
+      "token": "eyJhbGciOiJSUzI1NiIs..."
+    }
+    ```
```

Comment:
Maybe note that the token can be decoded by any JWT decoding tool to see the license information